spec/lib/rspec/retry_spec.rb

```diff
@@ -25,9 +38,11 @@ def count_up
     @count += 1
   end
 
+  # rubocop:disable Naming/AccessorMethodName
```

Comment:
IMO this is a bit cleaner if you put it inline on line 42, then you don't have to re-enable it.